.gitignore

```diff
@@ -263,3 +263,15 @@ typings/
 
 #WorkLogWeb front dependencies
 infrastructure/spring-based-app/work-log-web/src/main/
+onion-architecture.iml
```

Comment:
1. there is section for Intellij crap
2. use wildcards instead of including all files